### ISSUE TITLE
The library doesn't compile

### ISSUE THREAD

**Issue by artemstepanenko** [patchthecode/JTAppleCalendar#599]
The library doesn't compile anymore after I've updated to 7.1.1 (the most recent version). Xcode shows 56 issues under the `JSAppleCalendar` framework.
(To be more precise it started happening even before when @patchthecode suggested to switch to the master branch last week.)
Xcode 9.0, Swift 4.

**Comment by patchthecode** [comment]
What kind of errors are you seeing?

I just created a blank empty project, downloaded 7.1.1 and installed using cocoapods, and there is o errors.

Without knowing what kind of errors, i do not know much.
I am online for the day here -> https://gitter.im/patchthecode/JTAppleCalendar
This is my time zone -> https://www.timeanddate.com/time/zone/usa/california
If you message me too late, i wont be there.

**Comment by patchthecode** [comment]
After investigating with developer. 
It looks like a cocoapod issue.
Will wait to hear results.

**Comment by artemstepanenko** [comment]
There's something wrong with the version **1.3.1** of cocoapods, it doesn't generate a Podfile.lock for me. And when I tried to update JTAppleCaledar to the newest version (7.1.1) it downloaded files but didn't add new files to the Xcode project.
Once I downgraded cocoapods to **1.2.1** the issue was solved. Thus, it has nothing to do with the library itself.
@patchthecode thank you very much for the support.
